server/src/main/java/org/apache/druid/server/http/CoordinatorRedirectInfo.java

```diff
@@ -36,6 +38,11 @@ public class CoordinatorRedirectInfo implements RedirectInfo
       "/druid/coordinator/v1/isLeader"
   );
 
+  private static final Set<String> LOCAL_GET_PATHS = ImmutableSet.of(
+      "/druid-ext/basic-security/authentication/db/*",
```

Comment:
Some of these are cached, and some are backed by metadata store queries.

IMO, it would make sense to scale out the ones that are backed by metadata store queries if the concern is that the leader might be busy doing other things. However, to avoid stale reads, I wouldn't want to scale out the ones that are cached on the leader.

Did you have problems with some specific APIs in here? We can look at them individually.

Reply:
@gianm The biggest problem to be honest are the endpoints that are being polled periodically.
```
/druid-ext/basic-security/authentication|authorization/db/*/cachedSerializedUserMap
```
Other endpoints can probably be reverted if you don't think it's necessary.

Reply:
We are not worried about stale BasicAuth data because we usually configure authz of the cluster in the beginning when it was brand new and user addition after the fact is rare